src/MAUI/Maui.Samples/Platforms/Windows/Package.appxmanifest

```diff
@@ -46,4 +46,4 @@
 		<DeviceCapability Name="location" />
 		<DeviceCapability Name="bluetooth"/>
 	</Capabilities>
-</Package>
```

Comment:
why is this line change here?

Reply:
Not exactly able to figure out but it looks like GitHub changed line endings encoding after pushing the changes to repo. This change didn't appeared as part of the commit in Visual Studio.